### DISCUSSION TITLE
What is  External Data？

### DISCUSSION THREAD

**Discussion by wangxudong-cq** [onnx/onnx#3633] [Q&A]
https://github.com/onnx/onnx/blob/master/docs/PythonAPIOverview.md
The article mentions what is the External Data of Loading an ONNX Model with External Data？

**Comment by edgchen1** [comment]
There's some info here: https://github.com/onnx/onnx/blob/master/docs/IR.md#external-tensor-data

**Reply by jcwchen** [reply]
For how to use it and more details, you can check this: https://github.com/onnx/onnx/blob/master/docs/ExternalData.md
